server/routes/explore/api.py

```diff
@@ -118,6 +118,8 @@ def detect_and_fulfill():
       Params.EXP_MORE_DISABLED.value] = request.get_json().get(
           Params.EXP_MORE_DISABLED, "")
   utterance.insight_ctx[Params.DC.value] = dc_name
+  utterance.insight_ctx[Params.SKIP_RELATED_THINGS] = request.args.get(
```

Comment:
should here set the value to be a boolean while checking `lower(Params.SKIP_RELATED_THINGS.value) == 'true'`?

Reply:
done

Reply:
nit: use lower()?